src/course/views.py

```diff
@@ -134,7 +132,72 @@ def post(self):
                 current_user.courses.append(course)
                 models.db.session.commit()
                 author = models.User.query.filter_by(id=course.teacher_id).first()
-                print course.id
                 return "Redirect to:%s" % (url_for("view_course", courseID=course.id+1000))
         else:
             return "Registration Code Incorrect"
+
+class AddTAView(MethodView):
+
+    def get(self):
+        return
+
+    def post(self):
+        data = flask.request.get_json()
+        email = data.get('email')
+        courseId = data.get('courseID')
+        course = models.Course.query.filter_by(id=courseId).first()
+        if email:
+            user = models.User.query.filter(models.User.email.contains(email)).first()
+            if user:
```

Comment:
why isn't this just "if user and user.permissions and user.permissions <= 20"?
Also, why are we restricting being a TA to being a student?  Would it be complicated to allow authors to be TA's in some classes?